### PR TITLE
Add module meta

### PR DESCRIPTION
**Pull request by joneff**
and remove stray md files. extracted from #4285

### CODE REVIEW

File: packages/classic/scss/menu/_index.scss [hunk 1]
```diff
@@ -2,6 +2,17 @@
 @import "../utils/_index.scss";
 
 
+// Module meta
+$_kendo-module-meta: (
+    name: "menu-button",
```
Comment (TeyaVes) [inline packages/classic/scss/menu/_index.scss:7]: name should be menu

File: packages/classic/scss/menu-button/_index.scss [hunk 2]
```diff
@@ -2,6 +2,18 @@
 @import "../utils/_index.scss";
 
 
+// Module meta
+$_kendo-module-meta: (
+    name: "menu",
```
Comment (TeyaVes) [inline packages/classic/scss/menu-button/_index.scss:7]: name should be menu-button

File: packages/default/scss/menu-button/_index.scss [hunk 3]
```diff
@@ -2,6 +2,18 @@
 @import "../utils/_index.scss";
 
 
+// Module meta
+$_kendo-module-meta: (
+    name: "menu",
```
Comment (TeyaVes) [inline packages/default/scss/menu-button/_index.scss:7]: name should be menu-button
change dependencies with the correct ones for the menu-button

File: packages/default/scss/menu/_index.scss [hunk 4]
```diff
@@ -2,6 +2,17 @@
 @import "../utils/_index.scss";
 
 
+// Module meta
+$_kendo-module-meta: (
+    name: "menu-button",
```
Comment (TeyaVes) [inline packages/default/scss/menu/_index.scss:7]: - [ ] name should be menu
- [ ] change dependencies with the correct ones for the menu

**Review by TeyaVes — CHANGES_REQUESTED** [review]

File: packages/bootstrap/scss/menu-button/_index.scss [hunk 5]
```diff
@@ -2,6 +2,18 @@
 @import "../utils/_index.scss";
 
 
+// Module meta
+$_kendo-module-meta: (
+    name: "menu",
```
Comment (epetrow) [inline packages/bootstrap/scss/menu-button/_index.scss:7]: name should be menu-button and dependencies should be corrected to those of the menu-button

File: packages/bootstrap/scss/menu/_index.scss [hunk 6]
```diff
@@ -2,6 +2,17 @@
 @import "../utils/_index.scss";
 
 
+// Module meta
+$_kendo-module-meta: (
+    name: "menu-button",
```
Comment (epetrow) [inline packages/bootstrap/scss/menu/_index.scss:7]: name should be menu and dependencies should be corrected to those of the menu

**Review by epetrow — CHANGES_REQUESTED** [review]

**Pull request merged**